File: generate_theme.py
#!/usr/bin/env python
 
from random import choice

def main():
    with open('verbs.txt', 'r') as verbFile:
        verbs = verbFile.read().split()
    with open('nouns.txt', 'r') as nounFile:
        nouns = nounFile.read().split()

    for i in range(100):
        print '{verb} the {noun}'.format(verb=choice(verbs).title(), noun=choice(nouns).title())

if __name__ == "__main__":
    main()

File: generate_theme.pl
open(my $VERBS, "<", "verbs.txt") 
    or die "cannot open < verbs.txt: $!";
open(my $NOUNS, "<", "nouns.txt") 
    or die "cannot open < nouns.txt: $!";

@verbs = <$VERBS>; chomp @verbs;
@nouns = <$NOUNS>; chomp @nouns;

for ($count = 1; $count < 100; $count++) {
    $verb = ucfirst $verbs[int rand $#verbs];
    $noun = ucfirst $nouns[int rand $#nouns];
    print "$verb the $noun\n";
}

